onnx/defs/nn/defs.cc

```diff
@@ -98,7 +98,7 @@ The convolution operator consumes an input tensor and {filter_desc}, and
 computes the output.)DOC";
             ReplaceAll(doc, "{filter_desc}", filter_desc);
             schema.SetDoc(doc);
-            schema.NumInputs(2, 3);
+            schema.NumInputs(2);
```

Comment:
Can we keep NumInputs(2,3) for now? We'll need to make changes in caffe2 and pytorch side to make this working, otherwise it will break our tests. We can change it after the corresponding changes has been made.

Reply:
Sure.